### PR TITLE
cycode 0.2.5 (new formula)

### PR DESCRIPTION
**Pull request by MarshalX**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by MarshalX** [comment]
@p-linnane hello! I'm wondering is everything fine with PR? Thank you!

**Comment by MarshalX** [comment]
@samford, @chenrui333 @carlocab Hello, could you please take a look?

File: Formula/cycode.rb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+  resource "python-dateutil" do
+    url "https://files.pythonhosted.org/packages/4c/c4/13b4776ea2d76c115c1d1b84579f3764ee6d57204f6be27119f13a61d0a9/python-dateutil-2.8.2.tar.gz"
+    sha256 "0123cacc1627ae19ddf3c27a5de5bd67ee4586fbdd6440d9748f8abb483d3e86"
+  end
+
+  resource "pyyaml" do
+    url "https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz"
+    sha256 "68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2"
+  end
```
Comment (carlocab) [inline Formula/cycode.rb:90]: Replace this with
```ruby
depends_on "pyyaml"
```

File: Formula/cycode.rb [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+  resource "requests" do
+    url "https://files.pythonhosted.org/packages/9d/be/10918a2eac4ae9f02f6cfe6414b7a155ccd8f7f9d4380d62fd5b955065c3/requests-2.31.0.tar.gz"
+    sha256 "942c5a758f98d790eaed1a29cb6eefc7ffb0d1cf7af05c3d2791656dbd6ad1e1"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
```
Comment (carlocab) [inline Formula/cycode.rb:100]: Replace this with
```ruby
depends_on "six"
```

**Review by carlocab — COMMENTED** [review]
Hi there, thanks for your patience. This is looking good; just a few small suggestions.

File: Formula/cycode.rb [hunk 3]
```diff
@@ -0,0 +1,129 @@
+class Cycode < Formula
+  include Language::Python::Virtualenv
+
+  desc "Cycode CLI focuses on enhancing SDLC security by scanning for secrets, conducting Static Application Security Testing (SAST), Software Composition Analysis (SCA), and detecting Infrastructure as Code (IaC) misconfigurations, all while seamlessly integrating into developer workflows."
```
Comment (carlocab) [inline Formula/cycode.rb:4]: This description is too long.

Could you also squash your commits together, please? Thanks!

File: Formula/cycode.rb [hunk 4]
```diff
@@ -0,0 +1,129 @@
+class Cycode < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI boosts SDLC security via secret scanning, SAST, SCA, and IaC"
```
Comment (carlocab) [inline Formula/cycode.rb:4]:
```suggestion
  desc "Boost software development security via secret scanning, SAST, SCA, and IaC"
```
We should try to minimise the number of acronyms we use here.

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by MarshalX** [comment]
> Thanks!

Thank you! All checks have passed 🥳

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**